iocore/net/quic/QUICTypes.h

```diff
@@ -76,6 +76,13 @@ constexpr QUICEncryptionLevel QUIC_ENCRYPTION_LEVELS[] = {
   QUICEncryptionLevel::ONE_RTT,
 };
 
+// introduce by draft-19 kPacketNumberSpace
+enum class QUICPacketNumberSpace {
+  Initial,
+  Handshake,
+  ApplicationData,
+};
+
 // 0-RTT and 1-RTT use same Packet Number Space
 constexpr QUICEncryptionLevel QUIC_PN_SPACES[] = {
```

Comment:
Should we remove this in favor of `QUICPacketNumberSpace` ?

Reply:
will fix .

Reply:
fixed